Brokerages/Bitfinex/BitfinexBrokerage.cs

```diff
@@ -284,31 +284,51 @@ public override IEnumerable<BaseData> GetHistory(Data.HistoryRequest request)
                 yield break;
             }
 
+            if (request.StartTimeUtc >= request.EndTimeUtc)
+            {
+                throw new ArgumentException("BitfinexBrokerage.GetHistory(): Indvalid history request parameters, " +
```

Comment:
Thanks for adding the missing check 👍 
Just a typo in the message text: `Indvalid`